crates/sui-node/src/metrics.rs

```diff
@@ -187,8 +188,13 @@ pub fn start_metrics_push_task(config: &sui_config::NodeConfig, registry: Regist
 pub struct GrpcMetrics {
     inflight_grpc: IntGaugeVec,
     grpc_requests: IntCounterVec,
+    grpc_request_latency: HistogramVec,
 }
 
+const LATENCY_SEC_BUCKETS: &[f64] = &[
```

Comment:
nit: we could probably use a list similar to the one used in NW primary / worker latencies.

Reply:
I actually copy-pasted this directly from the NW one :)

want me to add a TODO to dedupe it somewhere?

Reply:
Hehe, we probably have several places with these in NW, I was referring to the (more comprehensive) one used by the primary and worker metrics: https://github.com/MystenLabs/sui/blob/main/narwhal/primary/src/metrics.rs#L17
We can upgrade it later if we see the need.

Reply:
Ohhh gotcha! Yeah I cribbed the ones from anemo metrics, I see what you mean about the more detailed ones. Sounds good.